internal/ngrokapi/edge_modules_https.go

```diff
@@ -57,57 +50,13 @@ func (c *defaultHTTPSEdgeModulesClientset) TLSTermination() *https_edge_tls_term
 	return c.tlsTermination
 }
 
-type EdgeRoutePolicyRawReplace struct {
-	EdgeID string          `json:"edge_id,omitempty"`
-	ID     string          `json:"id,omitempty"`
-	Module json.RawMessage `json:"module,omitempty"`
-}
-type RawHTTPSEdgePolicyClient interface {
-	Delete(context.Context, *ngrok.EdgeRouteItem) error
-	Replace(context.Context, EdgeRoutePolicyRawReplace) (*json.RawMessage, error)
-}
-type rawHTTPSPolicyClient struct {
-	base   *ngrok.BaseClient
-	policy *https_edge_route_policy.Client
-}
-
-func newRawHTTPSPolicyClient(config *ngrok.ClientConfig) *rawHTTPSPolicyClient {
-	return &rawHTTPSPolicyClient{
-		base:   ngrok.NewBaseClient(config),
-		policy: https_edge_route_policy.NewClient(config),
-	}
-}
-
-func (c *rawHTTPSPolicyClient) Delete(ctx context.Context, arg *ngrok.EdgeRouteItem) error {
-	return c.policy.Delete(ctx, arg)
-}
-
-func (c *rawHTTPSPolicyClient) Replace(ctx context.Context, policy *EdgeRoutePolicyRawReplace) (*json.RawMessage, error) {
-	if policy == nil {
-		return nil, errors.New("edge route policy replace cannot be nil")
-	}
-	var path bytes.Buffer
-	if err := template.Must(template.New("replace_path").Parse("/edges/https/{{ .EdgeID }}/routes/{{ .ID }}/policy")).Execute(&path, policy); err != nil {
-		// api client panics on error also
-		panic(err)
-	}
-	var res json.RawMessage
-	apiURL := &url.URL{Path: path.String()}
-
-	if err := c.base.Do(ctx, "PUT", apiURL, policy.Module, &res); err != nil {
-		return nil, err
-	}
-	return &res, nil
-}
-
 type HTTPSEdgeRouteModulesClientset interface {
 	Backend() *https_edge_route_backend.Client
 	CircuitBreaker() *https_edge_route_circuit_breaker.Client
 	Compression() *https_edge_route_compression.Client
 	IPRestriction() *https_edge_route_ip_restriction.Client
 	OAuth() *https_edge_route_oauth.Client
-	Policy() *https_edge_route_policy.Client
-	RawPolicy() *rawHTTPSPolicyClient
```

Comment:
`RawPolicy` stuff was ripped out because we can't just shove the configured values directly into the API client anymore. It makes it cleaner to just use the provided client than to try to retrofit this.